src/log.c

```diff
@@ -518,6 +544,7 @@ void qd_log_initialize(void)
     default_log_source->includeTimestamp = true;
     default_log_source->includeSource = 0;
     default_log_source->sink = log_sink_lh(SINK_STDERR);
+    default_log_source->add_log_stats = false;
```

Comment:
This line might not be necessary because the call to qd_log_source() already sets the add_log_stats flag to false.

Reply:
Leave it. It's a nanosecond.

Reply:
And anyway -- this function initializes the log. To initialize the log, that flag should be set to false.

Reply:
If you look at https://github.com/apache/qpid-dispatch/blob/main/src/log.c#L368, the ZERO(log_source); is called. This sets all bits to 0 on the log_source which then automatically sets the add_log_stats to false, so no need to set the add_log_stats explicitly to false in qd_log_source_defaults() which is called right after the call to ZERO(log_source); 
But yes, if ZERO(log_source);  was not called in the future, then qd_log_source_defaults() would take care of defaulting the add_log_stats flag.

Reply:
I wonder if the call to ZERO should be moved inside the qd_log_source_defaults() function. If we do that, the qd_log_source_defaults() can be called from the place where ZERO is currently being called from. And then, there would be no need to set the add_log_stats to false.